lib/Parse/ParseIfConfig.cpp

```diff
@@ -53,6 +53,17 @@ static StringRef extractExprSource(SourceManager &SM, Expr *E) {
   return SM.extractText(Range);
 }
 
+static bool isValidPrefixUnaryOperator(llvm::Optional<StringRef> &UnaryOperator) {
```

Comment:
Nitpick: We import `Optional` into the `swift` namespace like Clang does, so you can just use `Optional<StringRef>` here. Probably no need to pass by reference either.